proxy_pagination/__init__.py

```diff
@@ -16,6 +16,14 @@ class ProxyPagination(BasePagination):
     mapping = {alias: import_from_string(path, 'PROXY_PAGINATION_MAPPING')
                for alias, path in settings.PROXY_PAGINATION_MAPPING.items()}
 
+    @property
+    def display_page_controls(self):
+        return self.pager.display_page_controls
+
+    @property
+    def template(self):
+        return self.pager.template
+
```

Comment:
Ok now it looks fine, but Travis CI complains about whitespace in empty lines 26 and 30.